### ISSUE TITLE
Queries about the MAML implementation

### ISSUE THREAD

**Issue by patrolli** [oscarknagg/few-shot#29]
In line 84 few_shot/maml.py
`loss.backward(retain_graph=True)`
I don't understand the usage of this sentence. I wonder it aims to calculate the gradient of task_val_loss towards the initial parameters, but it has not been used in the latter codes. Could you explain the meaning of this sentence?  Thanks a lot!

**Comment by frajem** [comment]
I'm interested to know the answer to this question, too. Thank you!

**Comment by atseng17** [comment]
I wonder if this line is not necessary as there are no updates on the weights after this line, but I might be wrong
